assets/airports/eddl.json

```diff
@@ -0,0 +1,3131 @@
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["DL050", "DL053", "NIKOG", "LUSIX", "DL054", "SONEB"]
+            ]
+        },
+        "SONEB5Z": {
+            "icao": "SONEB5Z",
+            "name": "Soneb Five Zulu",
+            "rwy": {
+                "EDDL05R": []
+            },
+            "body": ["DL050", "DL053", "NIKOG", "LUSIX", ["DL054", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["SONEB"]
+            ]
+        },
+        "SONEB7T": {
+            "icao": "SONEB7T",
+            "name": "Soneb Seven Tango",
+            "rwy": {
+                "EDDL23L": []
+            },
+            "body": ["^DL230", ["DL235", "S190-"], "DL248", "ERKUM", ["DL245", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["SONEB"]
+            ]
+        }
+    },
+    "stars": {
+        "ABIGI1A": {
+            "icao": "ABIGI1A",
+            "name": "Abigi One Alpha",
+            "entryPoints": {
+                "ABIGI": [["ABIGI", "A70+"]]
+            },
+            "body": ["DL505", "DL556", ["DIKMI", "A30+"]],
+            "rwy": {
+                "EDDL05L": []
+            },
+            "draw": [
+                ["ABIGI", "DL505", "DL556"]
+            ]
+        },
+        "ABIGI1B": {
+            "icao": "ABIGI1B",
+            "name": "Abigi One Bravo",
+            "entryPoints": {
+                "ABIGI": [["ABIGI", "A70+"]]
+            },
+            "body": ["DL505", "DL555", ["NATOS", "A30+"]],
+            "rwy": {
+                "EDDL05R": []
+            },
+            "draw": [
+                ["ABIGI"]
+            ]
+        },
+        "BABUV1A": {
+            "icao": "BABUV1A",
+            "name": "Babuv One Alpha",
+            "entryPoints": {
+                "BABUV": [["BABUV", "A70+"]]
+            },
+            "body": ["DL505", "DL556", ["DIKMI", "A30+"]],
+            "rwy": {
+                "EDDL05L": []
+            },
+            "draw": [
+                ["BABUV", "DL505"]
+            ]
+        },
+        "BABUV1B": {
+            "icao": "BABUV1B",
+            "name": "Babuv One Bravo",
+            "entryPoints": {
+                "BABUV": [["BABUV", "A70+"]]
+            },
+            "body": ["DL505", "DL555", ["NATOS", "A30+"]],
+            "rwy": {
+                "EDDL05R": []
+            },
+            "draw": [
+                ["BABUV"]
+            ]
+        },
+        "BIKMU05": {
+            "icao": "BIKMU05",
+            "name": "Bikmu Zero Five",
+            "entryPoints": {
+                "BIKMU": [["BIKMU", "A150-"]]
+            },
+            "body": ["RONAD", ["DL513", "S220"], "DL514", ["IBIKO", "S220"], ["GAPNU", "S220"], "DL522", "RONAD", "DL524", "DL525", ["DL530", "S220"], ["DL550", "S220"], "DL555"],
```

Comment:
Line 1529:
RONAD at 220kts

Reply:
Source?
![image](https://user-images.githubusercontent.com/19408472/42473445-8bf8b56e-83bc-11e8-9b08-1bc244054efb.png)
![image](https://user-images.githubusercontent.com/19408472/42473450-8fc869aa-83bc-11e8-907c-17488a703a13.png)

Reply:
Navigraph, don't worry about it though, doesn't make any difference and it looks like just a chart inconsistency